src/language-handlebars/printer-glimmer.js

```diff
@@ -649,6 +651,10 @@ function locationToOffset(source, line, column) {
   }
 }
 
+function doesNotHaveHashParams(node) {
+  return node.hash.pairs.length === 0;
```

Comment:
It is possible when node doesn't have `hash` or `hash.pairts` properites?

Reply:
According to glimmer `MustacheStatement` interface, this is not possible.
https://github.com/glimmerjs/glimmer-vm/blob/cba8afd3e665087ce9aa1457181b1955cfee91f0/packages/%40glimmer/syntax/lib/types/nodes.ts#L92
https://github.com/glimmerjs/glimmer-vm/blob/cba8afd3e665087ce9aa1457181b1955cfee91f0/packages/%40glimmer/syntax/lib/types/nodes.ts#L226